dev_config/config.yml.dist

```diff
@@ -28,14 +28,8 @@ mysql_host: 127.0.0.1
 # Specify mock order ID to skip order creation in MayoLINK
 ml_mock_order: pmitest
 
-# Specify MayoLINK URLs to point to non-production
+# Specify MayoLINK API endpoint and credentials
 # ml_orders_endpoint: https://test.orders.mayomedicallaboratories.com
```

Comment:
I think we should switch this endpoint from `mayomedicallaboratories` to `mayocliniclabs`

Reply:
Good idea, and good timing. Thanks!